jerry-core/parser/js/opcodes-dumper.h

```diff
@@ -16,25 +16,238 @@
 #ifndef OPCODES_DUMPER_H
 #define OPCODES_DUMPER_H
 
-#include "opcodes.h"
 #include "ecma-globals.h"
 #include "lexer.h"
+#include "opcodes.h"
+#include "serializer.h"
 
-typedef enum __attr_packed___
+/**
+ * Operand (descriptor of value or reference in context of parser)
+ */
+class jsp_operand_t
```

Comment:
Moving to C++ is still under discussion in #470 with no conclusion yet. This change would already start adding such features (i.e., classes).